.chloggen/issue-24291-random-attribute-value.yaml

```diff
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
This is technically a breaking change, in that currently passing `${random:0-100}` would pass the string literally but it would stop doing so. I think it is fine to break things here, but we need some sort of escaping syntax for passing the literal value.

Reply:
Good point. I will update it.